app/presenters/social_relay_presenter.rb

```diff
@@ -0,0 +1,24 @@
+class SocialRelayPresenter
+
+  def as_json options={}
+    {
+      "subscribe" => AppConfig.relay.inbound.subscribe,
+      "scope"     => AppConfig.relay.inbound.scope,
+      "tags"      => tags
+    }
+  end
+
+  def tags
+    return [] unless AppConfig.relay.inbound.scope == "tags"
+    if AppConfig.relay.inbound.pod_tags.present?
+      tags = AppConfig.relay.inbound.pod_tags.split(",")
+    else
+      tags = []
+    end
```

Comment:
Assignments to the same local in two branches always itches me a bit.

```rb
tags = if AppConfig.relay.inbound.pod_tags.present?
  AppConfig.relay.inbound.pod_tags.split(",")
else
  []
end

tags = []
tags.concat(AppConfig.relay.inbound.pod_tags.split(",")) if AppConfig.relay.inbound.pod_tags.present?
```
would be two more variants.